code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -95,8 +95,7 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 /datum/gas_mixture/turf/heat_capacity(data = MOLES)
 	var/list/cached_gases = gases
 	. = 0
-	for(var/id in cached_gases)
-		var/gas_data = cached_gases[id]
+	for(var/id,gas_data in cached_gases)
```

Comment:
```suggestion
	for(var/_id, gas_data in cached_gases)
```
_ to indicate it is unused in the loop, no standard on this but it's a rustism and I kinda like it